### PR TITLE
Standardize and rename standard projects, and move them to their respective directories.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/turmoil/TurmoilCardManifest.ts [hunk 1]
```diff
@@ -52,4 +53,9 @@ export const TURMOIL_CARD_MANIFEST = new CardManifest({
     {cardName: CardName.TERRALABS_RESEARCH, Factory: TerralabsResearch},
     {cardName: CardName.UTOPIA_INVEST, Factory: UtopiaInvest},
     {cardName: CardName.SEPTUM_TRIBUS, Factory: SeptumTribus},
-  ]});
+  ],
+  standardProjects: [
+    {cardName: CardName.BUFFER_GAS_STANDARD_PROJECT, Factory: BufferGasStandardProject},
```
Comment (sienmich) [inline src/cards/turmoil/TurmoilCardManifest.ts:58]: Buffer Gas comes from Prelude expansion, not Turmoil

File: src/cards/standardProjects/SellPatents.ts [hunk 2]
```diff
@@ -3,12 +3,12 @@ import {CardName} from '../../CardName';
 import {CardMetadata} from '../CardMetadata';
```
Comment (sienmich) [inline src/cards/standardProjects/SellPatents.ts:3]: There are now two Sell Patents copies (same with other SPs). Is it intentional?

File: src/cards/base/standardProjects/SellPatentsStandardProject.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import {Player} from '../../../Player';
```
Comment (sienmich) [inline src/cards/base/standardProjects/SellPatentsStandardProject.ts:1]: There are now two Sell Patents copies (same with other SPs). Is it intentional?

**Review by sienmich — CHANGES_REQUESTED** [review]

File: src/cards/turmoil/TurmoilCardManifest.ts (hunk 1 above)
Comment (sienmich) [reply src/cards/turmoil/TurmoilCardManifest.ts:58]: https://boardgamegeek.com/filepage/166087/terraforming-mars-prelude-stronghold-games-english

File: src/cards/standardProjects/SellPatents.ts (hunk 2 above)
Comment (kberg) [reply src/cards/standardProjects/SellPatents.ts:3]: This turned out to be a little sloppier than I expected. Give me some time to clean it up.

File: src/cards/base/standardProjects/SellPatentsStandardProject.ts (hunk 3 above)
Comment (kberg) [reply src/cards/base/standardProjects/SellPatentsStandardProject.ts:1]: My bad, this one should stay, the others should go.

File: src/cards/turmoil/TurmoilCardManifest.ts (hunk 1 above)
Comment (kberg) [reply src/cards/turmoil/TurmoilCardManifest.ts]: Whoops! Fixed.

File: src/CardName.ts [hunk 4]
```diff
@@ -286,6 +285,7 @@ export enum CardName {
     AERIAL_MAPPERS = 'Aerial Mappers',
     AEROSPORT_TOURNAMENT = 'Aerosport Tournament',
     AIR_SCRAPPING_EXPEDITION = 'Air-Scrapping Expedition',
+    AIR_SCRAPPING_STANDARD_PROJECT = 'Air Scrapping',
```
Comment (sienmich) [inline src/CardName.ts:288]: You moved Air Scrapping to the Venus part here, but the Buffer Gas and Colony are still at the top

File: src/CardName.ts [hunk 5]
```diff
@@ -1,14 +1,13 @@
...
-    STANDARD_ASTEROID = 'asteroid', // 'Asteroid' is already taken
-    STANDARD_AIR_SCRAPPING = 'Air Scrapping',
-    STANDARD_BUFFER_GAS = 'Buffer Gas',
-    STANDARD_BUILD_COLONY = 'Colony',
-    STANDARD_AQUIFER = 'Aquifer',
-    STANDARD_GREENERY = 'Greenery',
-    STANDARD_CITY = 'City',
+    SELL_PATENTS_STANDARD_PROJECT = 'Sell Patents',
+    POWER_PLANT_STANDARD_PROJECT = 'Power Plant:SP',
+    ASTEROID_STANDARD_PROJECT = 'Asteroid:SP',
```
Comment (sienmich) [inline src/CardName.ts:5]: I'm not sure if having some of them with ':SP' and some without is good idea.

**Review by sienmich — APPROVED** [review]
Looks good now. Just those two peeky things, feel free to ignore them

File: src/CardName.ts (hunk 4 above)
Comment (kberg) [reply src/CardName.ts:288]: The card names in this enum are all commingled, and then there started to be a separation by type. I'm not sure really what to do here. Perhaps throw caution to the wind and alphabetize the whole thing?

File: src/CardName.ts (hunk 5 above)
Comment (kberg) [reply src/CardName.ts:5]: It should be OK. You only need to :separator when when two cards have the same name.

**Comment by kberg** [comment]
Thanks!

**Pull request merged**